docs/product-assembly_pam-on-openshift/src/main/asciidoc/ba-openshift-overview-con.adoc

```diff
@@ -0,0 +1,35 @@
++
+* {CENTRAL} Monitoring is a web-based management and monitoring console. It can manage deployment of processes to {KIE_SERVERS} and provide performance monitoring, but does not include authoring capabilities. You can use this component to manage staging and production environments.
++
+* Smart Router is an optional layer between {KIE_SERVERS} and other components that interact with them. It is required if you want {CENTRAL} or {CENTRAL} Monitoring to interact with several different {KIE_SERVERS}. Also, when your environment includes many processes running on different {KIE_SERVERS}, Smart Router provides a single endpoint to all client applications. A client application can make a REST API call requiring any process; Smart Router automatically determines which {KIE_SERVER} must be called for any particular request.
+
+You can arrange these and other components into various environment configurations within OpenShift. You can use the templates provided with {PRODUCT} to deploy the most common combinations.
+
+The following environment types are typical:
+  
+* _Authoring_: an environment for creating and modifying processes using {CENTRAL}. It consists of pods that provide {CENTRAL} for the authoring work and a {KIE_SERVER} for test execution of the processes.
+* _Managed deployment_: an environment for running existing processes for staging and production purposes. This environment includes several groups of {KIE_SERVER} pods; you can deploy and undeploy processes on every such group and also scale the group up or down as necessary. Use {CENTRAL} Monitoring to deploy, run, and stop the processes and to monitor their execution. 
+* _Deployment with immutable servers_: an alternate environment for running existing processes for staging and production purposes. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a process or group of processes. You can not stop any process on the pod or add any new process to the pod. If you want to use another version of a process or modify the configuration in any other way, you create a new pod and displace the old one. In this system, you can use typical container-based integration workflows and do not need to use any other tools to manage the pods. Optionally, you can use {CENTRAL} Monitoring to monitor the performance of the environment and to stop and restart some of the process instances, but not to deploy additional processes to any {KIE_SERVER} or undeploy any existing ones.
+
+To deploy a {PRODUCT} environment on OpenShift, you can use the templates that are provided with {PRODUCT}. You can modify the templates to ensure that your environment suits your needs.
```

Comment:
I'd search all instances of "needs" and consider replacing with "requirements". Your call, just sounds odd to me.

Reply:
Honestly, it just goes against my sense of taste. "environment suits your requirements" sounds like doggerel poetry.